addon/components/au-toaster.hbs

```diff
@@ -0,0 +1,17 @@
+{{#if this.toaster.toasts.length}}
+<div class="au-c-toaster {{this.position}}">
+  {{#each this.toaster.toasts as |toast|}}
+    <AuAlert
+      @title={{toast.title}}
+      @skin={{toast.options.type}}
+      @icon={{toast.options.icon}}
+      @size="small"
+      @closable={{toast.options.closable}}
```

Comment:
I also noticed that if we don't set a `timeOut` and only the close button can be used, the toast object will never be removed from the list, (but it the alert will be hidden).

That doesn't seem ideal, but I don't think we can solve that unless we add something like a `onClose` action to the `AuAlert` component which would make it a "managed" component. I don't think it blocks the current PR though. It doesn't really affect the user I think.

Reply:
Yes, was aware of that but didn't know an easy/fast solution for that.